frontend/src/utils/TaskUtils.ts

```diff
@@ -383,3 +383,33 @@ export const unratedProductOwnerTasks: (
   });
   return unrated;
 };
+
+export const findMissingDevelopers = (
+  taskRatings: Taskrating[],
+  allUsers: RoadmapUser[],
+) => {
+  const ratingIds = taskRatings.map((rating) => rating.createdByUser);
+  return allUsers
+    ?.filter((user) => user.type === RoleType.Developer)
+    .filter((developer) => !ratingIds.includes(developer.id));
```

Comment:
The filters could be combined, and `allUsers` should not be undefined
```suggestion
    .filter((user) => user.type === RoleType.Developer && !ratingIds.includes(user.id));
```